Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -4438,14 +4438,15 @@ static void adjustSettingsForLockdownMode(Settings& settings, const WebPreferenc
 #if ENABLE(WEB_AUDIO)
     settings.setWebAudioEnabled(false);
 #endif
-    switch (settings.downloadableBinaryFontAllowedTypes()) {
-    case DownloadableBinaryFontAllowedTypes::Any:
-        settings.setDownloadableBinaryFontAllowedTypes(DownloadableBinaryFontAllowedTypes::Restricted);
+    switch (settings.downloadableBinaryFontTrustedTypes()) {
+    case DownloadableBinaryFontTrustedTypes::Any:
+        settings.setDownloadableBinaryFontTrustedTypes(DownloadableBinaryFontTrustedTypes::Restricted);
         break;
-    case DownloadableBinaryFontAllowedTypes::Restricted:
-    case DownloadableBinaryFontAllowedTypes::None:
+    case DownloadableBinaryFontTrustedTypes::Restricted:
+    case DownloadableBinaryFontTrustedTypes::None:
         break;
     }
+    settings.setExperimentalFontParserInLockdownMode(true);
```

Comment:
Why are we setting this to true here uncoditionally? Are we tryint to override the default behavior?

Reply:
I think I don't understand the enablement strategy for Lockdown Mode features. Right now there are 2 states:

* Lockdown Mode is off, fonts are parsed through the system font parser
* Lockdown Mode is on, fonts on the allowlist are parsed through the system font parser and the rest are parsed through the experimental font parser

Should I have another feature flag/another feature level so that we also get a third state? "Lockdown Mode is on, but the experimental font parser is disabled"